src/wazuh_modules/vulnerability_scanner/tests/unit/exclusionScanner_test.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Wazuh Vulnerability Scanner - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 21, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "exclusionScanner_test.hpp"
+#include "scanOrchestrator/exclusionScanner.hpp"
+#include "scanOrchestrator/scanContext.hpp"
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <vector>
+
+/*
+ * @brief Test instantion of the ExclusionScanner class.
```

Comment:
Typo: `instantiation`?
```suggestion
 * @brief Test instantiation of the ExclusionScanner class.
```
The same typo is repeated in the other test files.
And in the tests name: Test`Instantion`OfTheExclusionScannerClass, and in some comments

Reply:
done